backend/api/Areas/Property/Models/Search/GeoJsonFilterModel.cs

```diff
@@ -156,6 +162,12 @@ public class GeoJsonFilterModel : PageFilter
         /// <value></value>
         public int? ConstructionTypeId { get; set; }
 
+        /// <summary>
+        /// get/set - Building  name.
+        /// </summary>
+        /// <value></value>
+        public string? BuildingName { get; set; }
```

Comment:
Just filter on `Name`, then it'll work for either land or building.

Reply:
Also remove the `nullable`, as we're not using this new feature presently.  `string` are already nullable by nature.